components/sync15/src/request.rs

```diff
@@ -203,6 +216,8 @@ impl NormalResponseHandler {
     }
 }
 
+// This implementation puts the responses into arrays so they can
```

Comment:
rm. I do agree that the actual function itself could be better commented, but this comment belongs _inside_ the function, since it's not documentation for users of this function.